apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/shareEmbedModal/email-tab.tsx

```
@@ -0,0 +1,157 @@
+      }
+      toast.error(t("common.something_went_wrong_please_try_again"));
+    }
+  };
+
+  const renderTabContent = () => {
+    if (activeTab === "preview") {
+      return (
+        <div className="space-y-4 pb-4">
+          <div className="flex-1 overflow-y-auto rounded-lg border border-slate-200 bg-white p-4">
+            <div className="mb-6 flex gap-2">
+              <div className="h-3 w-3 rounded-full bg-red-500"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)